crates/vault-registry/src/lib.rs

```diff
@@ -1327,12 +1327,27 @@ impl<T: Config> Module<T> {
             })
             .collect::<Vec<(_, _)>>();
 
-        if vaults_with_issuable_tokens.is_empty() {
-            Err(Error::<T>::NoVaultWithIssuableTokens.into())
-        } else {
-            vaults_with_issuable_tokens.sort_by(|a, b| b.1.cmp(&a.1));
-            Ok(vaults_with_issuable_tokens)
-        }
+        vaults_with_issuable_tokens.sort_by(|a, b| b.1.cmp(&a.1));
```

Comment:
This change makes sense, but can you check that this won't break the UI/lib?

Reply:
I'll double-check, but there should already be handling. If there isn't I think it still makes more sense to fix the UI/lib to handle the empty array case than to throw an error here.